pyoanda/client.py

```diff
@@ -13,9 +13,8 @@
 
 from .exceptions import BadCredentials, BadRequest
 
-
-log = getLogger(__name__)
-
+logging.basicConfig(level='CRITICAL')
```

Comment:
Here you are defining a logger instead of getting the defined logger, we should use this approach in case of global logger doesn't exists.